functions/src/googleform-hook.ts

```diff
@@ -0,0 +1,115 @@
+import * as functions from 'firebase-functions'
+import * as admin from 'firebase-admin'
+
+import { Path } from './models/Path'
+import { StepType } from './models/StepType'
+import Logger from './utils/logger'
+
+function pathLabelToPathDocumentId(pathLabel: string): Path | null {
```

Comment:
refactor'in dong ke utils atau apa gitu biar bisa direuse di dua function win

Reply:
done